appstream.patch

```diff
@@ -0,0 +1,83 @@
+From a3dc26beae5b5d304f25fdd65c296df378bb3fd5 Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Matthias=20Mail=C3=A4nder?= <matthias@mailaender.name>
+Date: Sun, 18 Feb 2024 21:14:54 +0100
+Subject: [PATCH] Update AppStream metadata.
+
+---
+ build/CMakeLists.txt                          |  4 ++--
+ ...o.desktop => com.zerobrane.studio.desktop} |  0
+ ....xml => com.zerobrane.studio.metainfo.xml} | 23 +++++++++++++++----
+ 3 files changed, 20 insertions(+), 7 deletions(-)
+ rename zbstudio/res/{zbstudio.desktop => com.zerobrane.studio.desktop} (100%)
+ rename zbstudio/res/{zbstudio.appdata.xml => com.zerobrane.studio.metainfo.xml} (51%)
+
+diff --git a/build/CMakeLists.txt b/build/CMakeLists.txt
+index e80d1d5d5..32d61ab11 100644
+--- a/build/CMakeLists.txt
++++ b/build/CMakeLists.txt
+@@ -133,10 +133,10 @@ else()
+ 
+   # install .desktop file and hicolor icon theme
+   install(DIRECTORY ${TOPDIR}/zbstudio/res/icons/ DESTINATION share/icons/hicolor)
+-  install(FILES ${TOPDIR}/zbstudio/res/zbstudio.desktop DESTINATION share/applications)
++  install(FILES ${TOPDIR}/zbstudio/res/com.zerobrane.studio.desktop DESTINATION share/applications)
+ 
+   # install Linux software gallery metadata
+-  install(FILES ${TOPDIR}/zbstudio/res/zbstudio.appdata.xml DESTINATION share/appdata)
++  install(FILES ${TOPDIR}/zbstudio/res/com.zerobrane.studio.metainfo.xml DESTINATION share/metainfo)
```

Comment:
minimize the patches. Use the renames in the manifest.

Reply:
no. Because usually you add them, then when you add them you should do with the proper naming convention.

Here they already exist and we have facilities for that. Reduce the size of the patch.

Reply:
I initially created the file upstream.